interface/src/app/genes/gene-search.js

```diff
@@ -223,11 +223,14 @@ angular.module('tribe.gene.search', [
                 $scope.found = $scope.results.found;
                 var begin, end;
                 $scope.updatePage = function(page) {
-                    begin = ((page-1)*3), end=begin+3;
```

Comment:
This is the main change on this file. All the others are coding style tweaking, mostly done by emacs editor automatically.